src/components/layout/BodyLayout.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <main>
-    <BotAlerts />
+    <BotAlerts style="position: absolute; top: 10px; right: 10px; width: 480px; z-index: 100" />
```

Comment:
I'm not sure we want the allerts to be hovering? (like a toast) ... ? 

If we aim to change this, i'd want to look at this separately, not in the scope of a PR that's adding a completely unrelated feature.

Reply:
I do find the current version of alerts a bit jarring the way they mess with the overall layout, but sure it's not really in the scope of this PR. I can revert this and changing them can be discussed in a separate PR?

Reply:
Absolutely - it can be a great enhancement for a separate PR (either in parallel or later) - but i don't like scope-creep, where random things are modifiedas part of a PR - it draws attention from the actual change away to tons of other things, leading to missing things.